.github/workflows/deploy-experimental.yml

```diff
@@ -156,6 +156,7 @@ jobs:
           pushd solution/backend
           npm install serverless -g
           npm install
+          serverless deploy --config ./serverless-redirect.yml --stage redirect${PR} | tee output.log
           serverless deploy --config ./serverless-experimental.yml --stage dev${PR} | tee output.log
           serverless invoke --config ./serverless-experimental.yml --function create_database --stage dev${PR}
```

Comment:
Will remove this after we deploy to prod. This was just to test it out on experimental.